system/duplicated_node_checker/config/duplicated_node_checker.param.yaml

```diff
@@ -2,3 +2,5 @@
   ros__parameters:
     update_rate: 10.0
     add_duplicated_node_names_to_msg: true # if true, duplicated node names are added to msg
+    nodes_to_ignore:  # List of nodes to ignore when checking for duplicates
+      - /rviz
```

Comment:
In Autoware, the node name of rviz is  ```/rviz2```.
https://github.com/tier4/autoware_launch/blob/tier4/main/autoware_launch/launch/autoware.launch.xml#L129

Reply:
Good catch, thank you! fixed here: https://github.com/autowarefoundation/autoware.universe/pull/7959/commits/945beea1f80a8c6459d71376bb09da5aecf5332d